qiskit/primitives/__init__.py

```diff
@@ -387,14 +387,15 @@
 Primitives API
 ==============
 
-Primitives V2
+Estimator V2
 -------------
```

Comment:
Thanks for noticing that the title was wrong.

```suggestion
Estimator V2
------------
```